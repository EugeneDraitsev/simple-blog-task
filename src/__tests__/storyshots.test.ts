import initStoryShots from '@storybook/addon-storyshots'
import 'jest-styled-components'

initStoryShots({})
