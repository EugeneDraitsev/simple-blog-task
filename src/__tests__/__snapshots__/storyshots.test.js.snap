// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Popup Popup with animation 1`] = `
.c4 {
  position: fixed;
  display: inline-block;
  visibility: hidden;
  max-width: 80vw;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  background-color: rgb(255,255,255);
  border-radius: 2px;
  border: 1px solid rgb(0,0,0);
  padding: 20px;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: 20px;
}

.c6 {
  color: rgb(1,85,147);
  font-family: weld-light,sans-serif;
  font-size: 40px;
}

.c7 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c8 {
  background: url('') center center;
  cursor: pointer;
  width: 46px;
  height: 46px;
}

.c2 {
  display: inline-block;
  opacity: 0;
  visibility: hidden;
  position: fixed;
  top: 0;
  left: 0;
  background-color: rgba(0,0,0,0.5);
  width: 100vw;
  height: 100vh;
  -webkit-transition: opacity 0.3s;
  transition: opacity 0.3s;
}

.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  padding: 20px;
  color: rgb(1,85,147);
  font-family: weld-light,sans-serif;
  font-size: 40px;
  text-align: center;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  cursor: pointer;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c10 {
  width: 270px;
  height: 200px;
  background: url('test-image.jpg') center center;
}

.c11 {
  width: 160px;
  padding: 0 5px;
  text-align: center;
  font-size: 16px;
}

.c3 {
  -webkit-transform: translate(-100vw,-50%) scale(0);
  -ms-transform: translate(-100vw,-50%) scale(0);
  transform: translate(-100vw,-50%) scale(0);
  -webkit-transition: all 0.3s;
  transition: all 0.3s;
}

<div
  className="c0"
>
  <div
    className="c1"
    onClick={[Function]}
  >
    Click to open popup
  </div>
  <div>
    <div
      className="c2"
      onClick={[Function]}
    />
    <div
      className="c3 c4"
    >
      <div
        className="c5"
      >
        <div
          className="c6"
        >
          test title
        </div>
        <div
          className="c7"
        />
        <div
          className="c8"
          onClick={[Function]}
        />
      </div>
      <div
        className="greeting"
      >
        <div
          className="c9"
        >
          <div
            className="c10"
          />
          <div
            className="c11"
          >
            Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Popup Popup with overlay 1`] = `
.c3 {
  position: fixed;
  display: inline-block;
  visibility: hidden;
  max-width: 80vw;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  background-color: rgb(255,255,255);
  border-radius: 2px;
  border: 1px solid rgb(0,0,0);
  padding: 20px;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: 20px;
}

.c5 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c6 {
  background: url('') center center;
  cursor: pointer;
  width: 46px;
  height: 46px;
}

.c2 {
  display: inline-block;
  opacity: 0;
  visibility: hidden;
  position: fixed;
  top: 0;
  left: 0;
  background-color: rgba(0,0,0,0.5);
  width: 100vw;
  height: 100vh;
  -webkit-transition: opacity 0.3s;
  transition: opacity 0.3s;
}

.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  padding: 20px;
  color: rgb(1,85,147);
  font-family: weld-light,sans-serif;
  font-size: 40px;
  text-align: center;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  cursor: pointer;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c8 {
  width: 270px;
  height: 200px;
  background: url('test-image.jpg') center center;
}

.c9 {
  width: 160px;
  padding: 0 5px;
  text-align: center;
  font-size: 16px;
}

<div
  className="c0"
>
  <div
    className="c1"
    onClick={[Function]}
  >
    Click to open popup
  </div>
  <div>
    <div
      className="c2"
      onClick={[Function]}
    />
    <div
      className="c3"
    >
      <div
        className="c4"
      >
        <div
          className="c5"
        />
        <div
          className="c6"
          onClick={[Function]}
        />
      </div>
      <div
        className="greeting"
      >
        <div
          className="c7"
        >
          <div
            className="c8"
          />
          <div
            className="c9"
          >
            Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Popup Popup with title 1`] = `
.c3 {
  position: fixed;
  display: inline-block;
  visibility: hidden;
  max-width: 80vw;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  background-color: rgb(255,255,255);
  border-radius: 2px;
  border: 1px solid rgb(0,0,0);
  padding: 20px;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: 20px;
}

.c5 {
  color: rgb(1,85,147);
  font-family: weld-light,sans-serif;
  font-size: 40px;
}

.c6 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c7 {
  background: url('') center center;
  cursor: pointer;
  width: 46px;
  height: 46px;
}

.c2 {
  display: inline-block;
  opacity: 0;
  visibility: hidden;
  position: fixed;
  top: 0;
  left: 0;
  background-color: rgba(0,0,0,0.5);
  width: 100vw;
  height: 100vh;
  -webkit-transition: opacity 0.3s;
  transition: opacity 0.3s;
}

.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  padding: 20px;
  color: rgb(1,85,147);
  font-family: weld-light,sans-serif;
  font-size: 40px;
  text-align: center;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  cursor: pointer;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c9 {
  width: 270px;
  height: 200px;
  background: url('test-image.jpg') center center;
}

.c10 {
  width: 160px;
  padding: 0 5px;
  text-align: center;
  font-size: 16px;
}

<div
  className="c0"
>
  <div
    className="c1"
    onClick={[Function]}
  >
    Click to open popup
  </div>
  <div>
    <div
      className="c2"
      onClick={[Function]}
    />
    <div
      className="c3"
    >
      <div
        className="c4"
      >
        <div
          className="c5"
        >
          test title
        </div>
        <div
          className="c6"
        />
        <div
          className="c7"
          onClick={[Function]}
        />
      </div>
      <div
        className="greeting"
      >
        <div
          className="c8"
        >
          <div
            className="c9"
          />
          <div
            className="c10"
          >
            Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`Storyshots Popup Simple popup 1`] = `
.c3 {
  position: fixed;
  display: inline-block;
  visibility: hidden;
  max-width: 80vw;
  top: 50%;
  left: 50%;
  -webkit-transform: translate(-50%,-50%);
  -ms-transform: translate(-50%,-50%);
  transform: translate(-50%,-50%);
  background-color: rgb(255,255,255);
  border-radius: 2px;
  border: 1px solid rgb(0,0,0);
  padding: 20px;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin-bottom: 20px;
}

.c5 {
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
}

.c6 {
  background: url('') center center;
  cursor: pointer;
  width: 46px;
  height: 46px;
}

.c2 {
  display: inline-block;
  opacity: 0;
  visibility: hidden;
  position: fixed;
  top: 0;
  left: 0;
  background-color: rgba(0,0,0,0.5);
  width: 100vw;
  height: 100vh;
  -webkit-transition: opacity 0.3s;
  transition: opacity 0.3s;
}

.c0 {
  height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c1 {
  padding: 20px;
  color: rgb(1,85,147);
  font-family: weld-light,sans-serif;
  font-size: 40px;
  text-align: center;
  -webkit-text-decoration: underline;
  text-decoration: underline;
  cursor: pointer;
}

.c7 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c8 {
  width: 270px;
  height: 200px;
  background: url('test-image.jpg') center center;
}

.c9 {
  width: 160px;
  padding: 0 5px;
  text-align: center;
  font-size: 16px;
}

<div
  className="c0"
>
  <div
    className="c1"
    onClick={[Function]}
  >
    Click to open popup
  </div>
  <div>
    <div
      className="c2"
      onClick={[Function]}
    />
    <div
      className="c3"
    >
      <div
        className="c4"
      >
        <div
          className="c5"
        />
        <div
          className="c6"
          onClick={[Function]}
        />
      </div>
      <div
        className="greeting"
      >
        <div
          className="c7"
        >
          <div
            className="c8"
          />
          <div
            className="c9"
          >
            Lorem ipsum bla bla bla Lorem ipsum bla bla bla Lorem ipsum bla bla bla
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
